packages/react-native-reanimated/package.json

```diff
@@ -145,7 +145,7 @@
     "prettier": "^3.3.3",
     "react": "18.3.1",
     "react-native": "0.75.2",
-    "react-native-builder-bob": "^0.30.0",
+    "react-native-builder-bob": "patch:react-native-builder-bob@npm%3A0.30.0#~/.yarn/patches/react-native-builder-bob-npm-0.30.0-c300db1e73.patch",
```

Comment:
Let's bump to 0.30.2 (https://github.com/callstack/react-native-builder-bob/releases/tag/react-native-builder-bob%400.30.2)